utils/efushell/integrationtest.json

```diff
@@ -0,0 +1,16 @@
+
+{
+"Tests":	
+	[
+		{
+			"Module":"loki",
+			"Config":"./src/modules/loki/configs/STFCTestII.json",
+			"KafkaTopic":"loki_detector",
+			"Generator" : "loki_udp_generated",
+			"Packets" : 50,
```

Comment:
The test should check for the exact number of packets to be received by the efu.